doc/radosgw/adminops.rst

```diff
@@ -275,6 +275,9 @@ Get User Info
 
 Get user information.
 
+Either ``uid`` or ``access-key`` is required as a request parameter but ``uid`` is recommended as the required request parameter.
```

Comment:
This wording confuses me a bit.  Maybe

Either a ``uid`` or ``access-key`` must be supplied as a request parameter.  We recommend supplying ``uid``.

Reply:
That wording sounds good to me. I'll go ahead and update the PR